src/map/clif.c

```diff
@@ -1313,17 +1313,19 @@ static void clif_spawn_unit(struct block_list *bl, enum send_target target)
 	p.font = (sd) ? sd->status.font : 0;
 #endif
 #if PACKETVER >= 20120221
-	if (battle_config.show_monster_hp_bar && bl->type == BL_MOB && status_get_hp(bl) < status_get_max_hp(bl)) {
+	const struct mob_data *md = BL_UCCAST(BL_MOB, bl);
+	if (clif->show_monster_hp_bar(md,bl) == true) {
 		p.maxHP = status_get_max_hp(bl);
 		p.HP = status_get_hp(bl);
-	} else {
+	}
```

Comment:
Keep the `} else {` style as is, for more info on the project style you can read [Coding-Style](https://github.com/HerculesWS/Hercules/wiki/Coding-Style)

Reply:
Fixed in latest commit